app/utilities/schema.py

```diff
@@ -177,7 +178,44 @@ def load_schema_from_url(schema_url, language_code):
 
     constructed_schema_url = f"{schema_url}?language={language_code}"
 
-    req = requests.get(constructed_schema_url)
+    session = requests.Session()
+
+    retries = Retry(
+        total=3,
+        allowed_methods=frozenset({"GET"}),
```

Comment:
This doesn't need to be a frozenset?. Also does this have a default?.

Reply:
Defaults are `DEFAULT_ALLOWED_METHODS = frozenset({'DELETE', 'GET', 'HEAD', 'OPTIONS', 'PUT', 'TRACE'})`. "By default, we only retry on methods which are considered to be idempotent (multiple requests with the same parameters end with the same state)". Do we want to only keep `GET`?

Reply:
Removed.